kore/src/main/scala/org/kframework/collections.scala

```diff
@@ -19,7 +19,6 @@ object Collections {
   def immutable[T](s: java.util.Set[T]): Set[T]           = s.asScala.toSet
   def immutable[T](s: java.util.List[T]): Seq[T]          = s.asScala
   def immutable[K, V](s: java.util.Map[K, V]): Map[K, V]  = s.asScala
-  def immutable[T](s: Array[T]): Seq[T]                   = s
```

Comment:
Removed this because it overlaps with the `immutable[T](s: java.util.List[T]): Seq[T]` overload, making the latter unusable